packages/contracts/src/dollar/libraries/LibCreditNftManager.sol

```diff
@@ -34,36 +34,36 @@ library LibCreditNftManager {
         );
 
     /// @notice Emitted when Credit NFT to Governance conversion rate was updated
-    event ExpiredCreditNFTConversionRateChanged(
+    event ExpiredCreditNftConversionRateChanged(
         uint256 newRate,
         uint256 previousRate
     );
 
     /// @notice Emitted when Credit NFT block expiration length was updated
-    event CreditNFTLengthChanged(
-        uint256 newCreditNFTLengthBlocks,
-        uint256 previousCreditNFTLengthBlocks
+    event CreditNftLengthChanged(
+        uint256 newCreditNftLengthBlocks,
+        uint256 previousCreditNftLengthBlocks
     );
 
     /// @notice Struct used as a storage for the current library
-    struct CreditNFTMgrData {
+    struct CreditNftMgrData {
         //the amount of dollars we minted this cycle, so we can calculate delta.
         // should be reset to 0 when cycle ends
         uint256 dollarsMintedThisCycle;
         uint256 blockHeightDebt;
-        uint256 creditNFTLengthBlocks;
-        uint256 expiredCreditNFTConversionRate;
+        uint256 creditNftLengthBlocks;
+        uint256 expiredCreditNftConversionRate;
         bool debtCycle;
     }
 
     /**
      * @notice Returns struct used as a storage for this library
      * @return l Struct used as a storage
      */
-    function creditNFTStorage()
+    function creditNftStorage()
         internal
         pure
-        returns (CreditNFTMgrData storage l)
```

Comment:
I wish we did not shorten "manager", instead it should be `CreditNftManagerData`